sql-plugin/src/main/321cdh/scala/com/nvidia/spark/rapids/shims/spark321cdh/SparkShimServiceProvider.scala

```diff
@@ -14,14 +14,14 @@
  * limitations under the License.
  */
 
-package com.nvidia.spark.rapids.shims.spark311cdh
+package com.nvidia.spark.rapids.shims.spark321cdh
 
 import com.nvidia.spark.rapids.{ClouderaShimVersion, ShimVersion}
 
 object SparkShimServiceProvider {
-  val VERSION = ClouderaShimVersion(3, 1, 1, "3.1.7270")
-  // cdh version can have numbers after after 7270
-  val CDH_BASE_VERSION = "3.1.1.3.1.7270"
+  val VERSION = ClouderaShimVersion(3, 2, 1, "3.2.7170")
+  // cdh version can have numbers after after 7170
+  val CDH_BASE_VERSION = "3.2.1.7170.1000"
```

Comment:
can we make this match the version you have in the pom.xml file: 3.2.1.3.2.7171000.0-3

if there is some issue with that perhaps we need to make the matchesVersion function below different

Reply:
fixed in latest commit